quipucords/api/migrations/0003_system_platform_id.py

```diff
@@ -9,17 +9,23 @@
 
 from api.deployments_report.serializer import SystemFingerprintSerializer
 
+CANONICAL_FACTS = ['bios_uuid', 'etc_machine_id', 'insights_client_id',
```

Comment:
I think we should just reference the other list.  We may have to move them to a location that both can import.